index.html

```diff
@@ -60,10 +61,10 @@ <h1>I’m Nyamekye. <br>
 
   <!--- Start of cards container -->
   <section class="work" id="portfolio">
-    <div class="container">
-      <div class="card">
+    <div class="container wrapper">
+      <div class="card" id="1">
         <div class="card-left">
-          <img src="images/work/Snapshoot Portfolio.png" alt="snap-card">
+          <img src="images/desktop-switch/(1)Snapshoot desktop.png" alt="snap-card">
         </div>
         <div class="card-right">
```

Comment:
- [ ] It is required that already created HTML code for the projects list should be removed, and instead it should be added via JavaScript. kIndly remove all the code for projects list here and create them dynamically on page load from your javascript file.
- [ ] Both the projects list on the page and the popup window must be created dynamically via javascript.


### Project Requirement
![project-rqt](https://github.com/evansnyamekye/Portfolio-CompleteWebsite/assets/26597432/7c674db7-008c-4ad9-a67c-432a9dc280d0)